panopticum/views.py

```diff
@@ -101,22 +137,6 @@ def dashboard_components(request):
     return render(request, 'dashboard/components.html')
 
 
-def dashboard_operations(request):
```

Comment:
It seems you are going to remove my patches

Reply:
yep. Some rebase merge with master mistakes